File: 231228/특정 원소들의 합/sum-of-specific-elements.py
arr = [list(map(int,input().split()))for _ in range(4)]

row,col = 0,0
tmp = 0
while 1:
    for i in range(col+1):
        tmp+=arr[row][i]
    row+=1
    col+=1

    if col==4:
        print(tmp)
        break

File: 240622/특정 원소들의 합/sum-of-specific-elements.js
const fs = require('fs')
const input = fs.readFileSync(0).toString().trim().split('\n')

let ans = 0
for(let i=0;i<4;i++){
    let arr = input[i].split(' ').map(Number)
    for(let j=0;j<i+1;j++){
        ans+=arr[j]
    }
}

console.log(ans)